hr_employee_relative/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows storing information about employee's family.
```

Comment:
Can you add a note about hr_family which provides a similar feature with a different implementation?

Reply:
Absolutely, what text/link should I add?

Reply:
But why maintaining `hr_family` if this one is more complete?